x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -32,13 +32,22 @@ const navigationTree: NavigationTreeDefinition = {
           title: i18n.translate('xpack.serverlessObservability.nav.discover', {
             defaultMessage: 'Discover',
           }),
-          link: 'discover',
+          link: 'observability-log-explorer',
+          // prevent this entry from ever becoming active, effectively falling through to the obs-log-explorer child
+          getIsActive: () => false,
+          // avoid duplicate "Discover" breadcrumbs
+          breadcrumbStatus: 'hidden',
           renderAs: 'item',
           children: [
             {
-              // This is to show "observability-log-explorer" breadcrumbs when navigating from "discover" to "log explorer"
-              link: 'observability-log-explorer',
+              link: 'discover',
               sideNavStatus: 'hidden',
+              children: [
+                {
+                  link: 'observability-log-explorer',
+                  sideNavStatus: 'hidden',
```

Comment:
You don't need the `sideNavStatus` prop. Neither on L44.
That's because you already declared `renderAs: 'item'` which means that none of the children will be rendered anyway.

Reply:
Thanks for that hint :+1: I removed the lines.